index.html

```diff
@@ -153,7 +153,7 @@
             <hr>
             <a class="dropdown-item" href="/docs/c_glib">C GLib</a>
             <a class="dropdown-item" href="/docs/cpp">C++</a>
-            <a class="dropdown-item" href="https://github.com/apache/arrow/blob/main/csharp/README.md" target="_blank" rel="noopener">C#</a>
+            <a class="dropdown-item" href="https://github.com/apache/arrow-dotnet" target="_blank" rel="noopener">C#</a>
```

Comment:
```suggestion
            <a class="dropdown-item" href="https://github.com/apache/arrow-dotnet" target="_blank" rel="noopener">.NET</a>
```

Could you sort this list with ".NET"?

Before:
* C GLib
* C++
* C#
* ...

After:
* .NET
* C GLib
* C++
* ...